cmd/mothership/mothership/start/test/requests/new_occupancy.json

```diff
@@ -0,0 +1,5 @@
+{
+  "component": "component",
```

Comment:
Naming of the field is a bit confusing.. its' actually the name of the component-reconciler and not of the component (e.g. the component reconciler `base` is used for multiple components).